sdk/textanalytics/ai-text-analytics/sample.env

```diff
@@ -14,4 +14,4 @@ AZURE_CLIENT_SECRET=<client secret used to authenticate to Azure AD>
 
 # Our tests assume that TEST_MODE is "playback" by default. You can
 # change it to "record" to generate new recordings, or "live" to bypass the recorder entirely.
-# TEST_MODE=playback
\ No newline at end of file
+# TEST_MODE=live
```

Comment:
Nitpick: this seems odd. Since this line is commented out, it won't actually do anything and we want to indicate that playback is the default mode here.

Reply:
done.